kmath-geometry/src/commonMain/kotlin/scientifik/kmath/geometry/Line.kt

```diff
@@ -1,6 +1,15 @@
 package scientifik.kmath.geometry
 
-data class Line<V: Vector>(val base: V, val direction: V)
+import scientifik.kmath.dimensions.D2
+import scientifik.kmath.dimensions.D3
+import scientifik.kmath.dimensions.Dimension
 
-typealias Line2D = Line<Vector2D>
-typealias Line3D = Line<Vector3D>
+// TODO think how we can add asserts about direction.norm() > 0
```

Comment:
I'm sure if it should be explicitly checked

Reply:
can you please elaborate?